docker/Dockerfile.cross

```diff
@@ -0,0 +1,7 @@
+FROM rust:latest
+
+ENV CROSS_DOCKER_IN_DOCKER=true
+
+RUN wget -O- https://download.docker.com/linux/static/stable/x86_64/docker-20.10.9.tgz | tar -C /usr/bin/ --strip-components=1 -xzf - docker/docker
```

Comment:
We need to check signature or checksum here so we don't rely on the file being unmodified in future.

Reply:
I don't think this is necessary.

Reply:
It is not rare for tarball files to change behind the scene, so ensuring it is valid is a good security matter.

Reply:
Is there a Docker image we can copy the binary from instead using a multi-stage build?

Reply:
There is a [docker in docker image](https://hub.docker.com/_/docker,) but it has some other binaries which are not necessary and rust has to be installed manually which takes more steps than this multi-stage build.
The docker in docker image receives it binaries from `download.docker.com` ([layer 6](https://hub.docker.com/layers/docker/library/docker/latest/images/sha256-3e1787157b0cd05a6c56f1d7bc916aa486f53f40f0d1538e34b917bdf947f94d)) just like here, so there is no big difference when using the docker in docker image (besides the unwanted binaries which it has).